### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by tasnhubaislam**

### CODE REVIEW

**Review by joerossi215 — APPROVED** [review]
Looks great. Nice Work!

**Review by yellow-tulips — APPROVED** [review]
I approve of these changes

File: index.php [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<?php 
+    session_start();
```
Comment (Taraque-A) [inline index.php:2]: Great way to start!

File: index.php [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+	}
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Taraque-A) [inline index.php:39]: Looks nice!

File: index.php [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                <div id="quote"> 
+                    <div id="output">Get your quote of the day!</div>
+                    <button onclick="quotes()" id = "button"> click me!<br></button>
+                </div>
+            </div>
+
+
+            <div class = "goalTracker">
+                <div class = "tracker">
+                    <h2>Goal Tracker</h2>
```
Comment (Taraque-A) [inline index.php:124]: Really great idea!

File: login.php [hunk 4]
```diff
@@ -0,0 +1,70 @@
+<?php 
+    session_start();
```
Comment (Taraque-A) [inline login.php:2]: Nice!

File: login.php [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  <title>Login</title>
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="stylelogin.css">
+</head>
+<body>
+    <header>
+        <img src="logo.png" width="150" height="120">
+    </header>
+  <div class="wrapper">
+    <h1>Hello Again!</h1>
```
Comment (Taraque-A) [inline login.php:49]: This is look great!

**Review by farzanaisrat — APPROVED** [review]
looks good! good job!

**Review by Taraque-A — APPROVED** [review]

**Pull request closed without merging**